backend/database/models/officer.py

```diff
@@ -18,18 +16,17 @@ class Rank(str, enum.Enum):
     CHIEF = "CHIEF"
 
 
-class Officer(db.Model, SourceMixin):
+class Officer(db.Model):
     id = db.Column(db.Integer, primary_key=True)  # officer id
-    incident_id = db.Column(db.Integer, db.ForeignKey("incident.id"))
     first_name = db.Column(db.Text)
```

Comment:
It feels like `first_name`, `last_name`, `race`, `ethnicity`, `gender`, and `date_of_birth` could be pulled into a table called `individual` which `officer` and `victim` can both reference. This would allow for mapping of situations where an officer later became a victim.

`perpetrator` probably shouldn't have a reference to the `individual` table since it's providing information that would be used to identify suspected officers.

Reply:
Not sure we need that. The goal of the victim table is just to get demographic data, not to identify the victim as an individual. I want to avoid having any sort of database of victims identities.